README.md

```diff
@@ -6,11 +6,12 @@
 [![Open questions](https://img.shields.io/badge/Open-questions-pink.svg?style=flat-square)](https://github.com/theia-ide/theia/labels/question)
 [![Open bugs](https://img.shields.io/badge/Open-bugs-red.svg?style=flat-square)](https://github.com/theia-ide/theia/labels/bug)
 
-Theia is an extensible platform to develop full-fledged multi-language Cloud & Desktop IDE-like products with state-of-the-art web technologies.
+Theia is an extensible platform to develop full-fledged multi-language Cloud & Desktop IDE-like products with state-of-the-art web technologies. All languages which are implemented in the LSP [are supported](https://langserver.org/#implementations-server)
```

Comment:
Why should it be here? It is already covered in the scope. Also extensions should be implemented to support each language.

Reply:
The LSP support is very hidden, imho. I was not quite sure, what it means. 
This one is clean.

Reply:
There are a lot of different aspects which can be interested for different people. Putting all in one message does not sound good to me.

Maybe we can drop `Scope` section and put bullet points immediately after the main message as sub messages. We can iterate on them as well.

BTW The official LSP website: https://microsoft.github.io/language-server-protocol/ not one which is referenced

Reply:
Well, the thing is that language support is crucial for an editor. On what do you look first by an editor? 

If your languages are supported, I guess?

Because otherwise, the app makes no sense. 
I would even consider to reflect the full support matrix here, in order to provide this info for SEO.

Reply:
Let's please keep the description short as @akosyakov said. Having a bullet point list of main features right after it is good. Also, please use the official LSP site as a link.